end-from-dialog.js

```diff
@@ -47,6 +48,17 @@
 
   var queuedEndingNarration = null;
 
+  // Hook into game load and rewrite custom functions in game data to Bitsy format.
+  var _load_game = load_game;
+  globals.load_game = function(game_data, startWithTitle) {
+     // Rewrite custom functions' parentheses to curly braces for Bitsy's
+     // interpreter. Unescape escaped parentheticals, too.
+    var fixedGameData = game_data
+      .replace(/(^|[^\\])\(((end|endNow)( ".+?")?)\)/g, "$1{$2}") // Rewrite (end...) to {end...}
+      .replace(/(^|\\)\(((end|endNow)( ".+?")?)\\?\)/g, "($2)");   // Rewrite \(end...\) to (end...)
```

Comment:
this will also pick up `\(end)` but not `(end\)` btw

Reply:
Yeah, that's intentional. If you're lazy, you can just escape the opening parenthesis. It's weird if you only escape the closing one.